collector/meminfo_bsd.go

```diff
@@ -22,30 +22,63 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+type sysctlType uint8
+
+const (
+	t_uint32 sysctlType = iota
```

Comment:
Nit: Go names shouldn't have underscores in them. If the type is `sysctlType`, it should likely be something like `sysctlTypeUint32` (even though that's more verbose).

Same for `t_uint64`.